src/main/java/com/dev/cinema/model/dto/cart/ShoppingCartDtoMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.dev.cinema.model.dto.cart;
+
+import com.dev.cinema.model.ShoppingCart;
+import com.dev.cinema.model.Ticket;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartDtoMapper {
+    public ShoppingCartResponseDto mapToResponseDto(ShoppingCart shoppingCart) {
+        ShoppingCartResponseDto cartResponseDto = new ShoppingCartResponseDto();
+        cartResponseDto.setCartId(shoppingCart.getUser().getId());
```

Comment:
```suggestion
        cartResponseDto.setCartId(shoppingCart.getId());
```
they hold the same values, but anyways

Reply:
done